docs/cli-spec.md

```diff
@@ -0,0 +1,47 @@
+# Signing CLI specficiation
+
+### Inputs
+
+The parameters to the signing client are as follows:
+
+`sign code AzureKeyVault [options] <files>`
+
+Options:
+
+- `-o | --output` - The output file. If omitted, overwrites input. Must be a directory if multiple files are specified.
+- `-b | --base-directory` - Base directory for files to override the working directory.
+- `-f | --force` - Overwrites a signature if it exists.
+- `-m | --max-concurrency` - Maximum concurrency (default is 4)
+- `-fl | --filelist` - Path to file containing paths of files to sign within an archive
+- `-fd | --file-digest` - The digest algorithm to hash the file with.
```

Comment:
I expected it would be enum, right? Which algorithm does it default to?
```suggestion
- `-fd | --file-digest` <HASHALGORITHM> - The digest algorithm to hash the file with.
```